modules/ocf/manifests/extrapackages.pp

```diff
@@ -114,15 +114,13 @@
     'php-curl',
     'php-gd',
     'php-mbstring',
-    'php-mcrypt',
     'php-mysql',
     'php-sqlite3',
     'php-soap',
     'php-xml',
     'php-zip',
     'php7.0-dba', # This isn't available as php-dba unfortunately
```

Comment:
php-dba is avaliable in buster now

Reply:
Bleh it's a virtual package though, which means puppet will try and install it on every run :/

Reply:
Ah I see you figured that out anyway, given https://github.com/ocf/puppet/commit/87e1a30e9d20e840029c986540f1b276d179e5ce